subworkflows/nf-core/vcf_annotate_ensemblvep_snpeff/tests/main.nf.test.snap

```diff
@@ -0,0 +1,338 @@
+                        "single_end": false
+                    },
+                    "snpEff_summary.html"
+                ],
+                [
+                    {
+                        "id": "test0_annotated",
+                        "single_end": false
+                    },
+                    "snpEff_summary.html"
+                ],
+                [
+                    {
+                        "id": "test1_annotated",
+                        "single_end": false
+                    },
+                    "snpEff_summary.html"
+                ]
+            ],
+            [
+                [
+                    {
+                        "id": "custom_test0_annotated",
+                        "single_end": false
+                    },
+                    "custom_test0_annotated.genes.txt"
+                ],
+                [
+                    {
+                        "id": "custom_test1_annotated",
+                        "single_end": false
+                    },
+                    "custom_test1_annotated.genes.txt"
+                ],
+                [
+                    {
+                        "id": "test0_annotated",
+                        "single_end": false
+                    },
+                    "test0_annotated.genes.txt"
+                ],
+                [
+                    {
+                        "id": "test1_annotated",
+                        "single_end": false
+                    },
+                    "test1_annotated.genes.txt"
+                ]
+            ],
+            [
+                "versions.yml",
+                "versions.yml",
+                "versions.yml",
+                "versions.yml",
+                "versions.yml",
+                "versions.yml",
+                "versions.yml"
+            ]
+        ],
+        "meta": {
+            "nf-test": "0.9.0",
+            "nextflow": "24.04.4"
+        },
+        "timestamp": "2024-10-18T07:57:12.127577423"
+    },
+    "sarscov2 - ensemblvep - large chunks": {
+        "content": [
+            [
+                [
+                    {
+                        "groupSize": 1,
+                        "groupTarget": {
+                            "id": "custom_test",
+                            "single_end": false
+                        }
+                    },
+                    "custom_test.vcf.gz,variantsMD5:370b92bc4c236a7b4c6d152319447af3",
+                    "custom_test.vcf.gz.tbi"
+                ]
+            ],
+            [
+                "custom_test0_annotated.vep.vcf.gz_summary.html"
+            ],
+            [
+                
+            ],
+            [
+                
+            ],
+            [
+                
+            ],
+            [
+                "versions.yml",
+                "versions.yml",
+                "versions.yml",
+                "versions.yml",
+                "versions.yml"
+            ]
```

Comment:
blanks ?

Reply:
Yes optional subwf outputs, I checked them and the correct ones are missing :p